enderio-base/src/main/java/com/enderio/base/common/recipe/capacitor/CapacitorDataRecipe.java

```diff
@@ -90,7 +92,7 @@ public ResourceLocation getId() {
         @Nonnull
         @Override
         public CapacitorDataRecipe fromJson(@Nonnull ResourceLocation recipeId, JsonObject json) {
-            Ingredient capacitor = Ingredient.fromJson(json.get("capacitor"));
+            Item capacitor = ForgeRegistries.ITEMS.getValue(new ResourceLocation(json.get("capacitor").getAsString())); // TODO: These may need more checks?
```

Comment:
just check if capacitor is null or the items registry name is different to the one in the json. If so, throw an exception, that doesn't crash the game, but will be logged and the recipe is disgarded